tensorflow/inference/docker/build_artifacts/sagemaker/python_service.py

```diff
@@ -276,8 +301,7 @@ def _setup_channel(self, grpc_port):
             log.info("Creating grpc channel for port: %s", grpc_port)
             self._channels[grpc_port] = grpc.insecure_channel("localhost:{}".format(grpc_port))
 
-    def _import_handlers(self):
-        inference_script = INFERENCE_SCRIPT_PATH
+    def _import_handlers(self, inference_script=INFERENCE_SCRIPT_PATH):
```

Comment:
Now that we're using this to load handlers from different paths I'd log the location of the handler in line 316

Reply:
Same as above. I will update this too.